app/services/big_blue_button_api.rb

```diff
@@ -51,8 +51,8 @@ def publish_recordings(record_ids:, publish:)
     bbb_server.publish_recordings(record_ids, publish)
   end
 
-  def update_recordings(record_id:, meta_hash:)
-    bbb_server.update_recordings(record_id, {}, meta_hash)
+  def update_recordings(record_ids:, meta_hash:)
```

Comment:
What's the point of this switch?

Reply:
To conform to the BBB-api-gem - it specifies record_ids and not record_id.

Reply:
Our service works separately from the gem. At the moment, there isn't any need to update multiple recordings at once, so I would leave the service as is

Reply:
Yeah we should - but we can do that in a separate PR